octoprint_octobullet/__init__.py

```diff
@@ -56,24 +156,30 @@ def on_settings_save(self, data):
 		octoprint.plugin.SettingsPlugin.on_settings_save(self, data)
 
 		import threading
-		thread = threading.Thread(target=self._connect_bullet, args=(self._settings.get(["access_token"]),
+		thread = threading.Thread(target=self._connect_bullet, args=(self._settings.get(["quiet_minutes"]),
+									     self._settings.get(["access_token"]),
 		                                                             self._settings.get(["push_channel"])))
 		thread.daemon = True
 		thread.start()
 
 	def get_settings_defaults(self):
 		return dict(
+			quiet_minute=0,
 			access_token=None,
 			push_channel=None,
 			printDone=dict(
 				title="Print job finished",
 				body="{file} finished printing in {elapsed_time}"
+			),
+			printPartial=dict(
+				title="Print job {progress}% complete",
+				body="{progress}% {file}\n{print_time}\n{remaining} left\n{ETA} finish"
 			)
 		)
 
 	def get_settings_restricted_paths(self):
 		# only used in OctoPrint versions > 1.2.16
-		return dict(admin=[["access_token"], ["push_channel"]])
+		return dict(admin=[["quiet_minutes"], ["access_token"], ["push_channel"]])
```

Comment:
I think we don't need to restrict access to "quiet_minutes", it's not really sensitive information, right?